internal/exec/engine.go

```diff
@@ -53,6 +53,7 @@ type Engine struct {
 	FetchTimeout      time.Duration
 	Logger            *log.Logger
 	Root              string
+	OEM               string
```

Comment:
I've changed a decent amount due to review since making this PR, I'll build/upload another image to amazon to test to make sure there's nothing else super nonsensical in here.